drivers/mfd/Kconfig.adp5585

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 NXP
+# SPDX -License-Identifier: Apache-2.0
```

Comment:
```suggestion
# SPDX-License-Identifier: Apache-2.0
```

Reply:
Resolved. Several other files have this same issue. You may open an issue to track it.

Reply:
thanks for the quick turnaround! and done #72454 :)